Cargo.toml

```diff
@@ -16,6 +16,9 @@ license = "MIT"
 
 description = "A contiguous region of bytes, useful for I/O operations."
 
+[dependencies]
+num = "*"
```

Comment:
Yeah that changes my "zero dependencies" into "rust-lang/num, rand, and rustc-serialize". That's not ok. I'll just pull in the trait myself.